ui.apps/src/main/content/jcr_root/apps/core/fd/components/commons/v1/menuTemplate/menu.html

```diff
@@ -0,0 +1,18 @@
+<template data-sly-template.menu>
+    <div data-cmp-hook-menu-template="includes">
+        <sly data-sly-use.clientlib="${'/libs/granite/sightly/templates/clientlib.html'}"
+             data-sly-call="${clientlib.all @ categories=['core.forms.components.commons.v1.clientlibs.menu']}"></sly>
+    </div>
+    <div class="cmp-portallister__menu" data-cmp-hook-menu-template="container">
+        <ul class="cmp-portallister__menu--list" data-cmp-hook-menu-template="list" role="menu">
+        </ul>
+    </div>
+    <script data-cmp-hook-menu-template="menuItemTemplate"
+            type="x-template" data-sly-call="${menuItem}">
+    </script>
+</template>
+<template data-sly-template.menuItem>
+    <li class="cmp-portallister__menu--item" data-cmp-hook-menu-template="item" role="menuitem" tabindex="0">
```

Comment:
This should be `cmp-portallister__menu-item` ... item is not a modifier here but rather than an element as per BEM